src/renderer/components/cluster-manager/clusters-menu.tsx

```diff
@@ -32,6 +32,9 @@ interface Props {
 @observer
 export class ClustersMenu extends React.Component<Props> {
   @observable workspaceMenuVisible = false;
+  @computed get workspace() {
```

Comment:
There is no need in `@computed` when nothing to calculate/derive inside the getter.

Reply:
Fair, will remove